default_python/src/notebook.ipynb

```diff
@@ -19,7 +19,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 0,
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2"
```

Comment:
Does this still work when using it from the job that references it?

And shouldn't this go into the `scratch` folder instead?

Reply:
We've used this in "Notebook Best Practices" so I don't expect this to be a problem with Jobs. It's incredibly useful for inner dev loop. I think it should be in all notebooks that we have in templates and examples.